app/main/models/enums.py

```diff
@@ -20,3 +20,46 @@ class PriorityType(enum.Enum):
     less_of_these = -1
     follow = 0
     more_of_these = 1
+
+
+class Month(enum.Enum):
+    jan = 0
+    feb = 1
+    mar = 2
+    apr = 3
+    may = 4
+    jun = 5
+    jul = 6
+    aug = 7
+    sep = 8
+    oct = 9
+    nov = 10
+    dec = 11
+
+    @classmethod
+    def get_month(cls, month):
```

Comment:
this is not required. https://stackoverflow.com/questions/41407414/convert-string-to-enum-in-python